### PR TITLE
Added a RAM indicator and setting to disable pause button

### PR DESCRIPTION
**Pull request by QuackingCanary**
Added a RAM indicator and setting to disable pause button on EmulationActivity. The RAM indicator will be shown at top right

### CODE REVIEW

File: app/src/main/java/emu/skyline/EmulationActivity.kt [hunk 1]
```diff
@@ -318,6 +320,17 @@ class EmulationActivity : AppCompatActivity(), SurfaceHolder.Callback, View.OnTo
                     }
                 }, 250)
             }
+
+            binding.ramStats.apply {
+                postDelayed(object : Runnable {
+                    override fun run() {
+                        updatePerformanceStatistics()
+                        ramUsage = File("/proc/self/statm").readLines()[0].split(' ')[1].toFloat() * 4096 / 1000000
```
Comment (lynxnb) [inline app/src/main/java/emu/skyline/EmulationActivity.kt:328]: Tiny comment for future readers
```suggestion
                        // We read the `VmRSS` value from the kernel
                        ramUsage = File("/proc/self/statm").readLines()[0].split(' ')[1].toFloat() * 4096 / 1000000
```

File: app/src/main/java/emu/skyline/EmulationActivity.kt [hunk 2]
```diff
@@ -318,6 +320,17 @@ class EmulationActivity : AppCompatActivity(), SurfaceHolder.Callback, View.OnTo
                     }
                 }, 250)
             }
+
+            binding.ramStats.apply {
+                postDelayed(object : Runnable {
+                    override fun run() {
+                        updatePerformanceStatistics()
```
Comment (lynxnb) [inline app/src/main/java/emu/skyline/EmulationActivity.kt:327]: You don't need to call this here.

File: app/src/main/java/emu/skyline/EmulationActivity.kt [hunk 3]
```diff
@@ -347,6 +360,11 @@ class EmulationActivity : AppCompatActivity(), SurfaceHolder.Callback, View.OnTo
 
         binding.onScreenPauseToggle.apply {
             isGone = binding.onScreenControllerView.isGone
+            if(!emulationSettings.showPauseButton) {
+                binding.onScreenPauseToggle.visibility = View.GONE
+            } else {
+                binding.onScreenPauseToggle.visibility = View.VISIBLE
+            }
```
Comment (lynxnb) [inline app/src/main/java/emu/skyline/EmulationActivity.kt:367]:
```suggestion
        binding.onScreenPauseToggle.apply {
            isGone = !emulationSettings.showPauseButton
        }
```

**Review by lynxnb — CHANGES_REQUESTED** [review]

File: app/src/main/java/emu/skyline/EmulationActivity.kt (hunk 2 above)
Comment (lynxnb) [reply app/src/main/java/emu/skyline/EmulationActivity.kt:327]: This is still there

Comment (QuackingCanary) [reply app/src/main/java/emu/skyline/EmulationActivity.kt:327]: How can I display the indicator to the right if I don't call ramStats here?

**Comment by AbandonedCart** [comment]
`git rebase -i HEAD~3 --preserve-merges`
press the down arrow, then the `s` key
change the pick to fixup for the second and third commits
press ecape key, then type`:x` and press enter
`git push -f`

This is assuming you are using a linux or mac OS

**Comment by QuackingCanary** [comment]
@AbandonedCart Thanks for your help. It says `error: cannot rebase: You have unstaged changes.`

**Comment by dima-xd** [comment]
> @AbandonedCart Thanks for your help. It says `error: cannot rebase: You have unstaged changes.`. Even though I don't have anything more to commit

probably broken submodules

**Comment by AbandonedCart** [comment]
`git status` will tell you what it thinks has changed

**Comment by AbandonedCart** [comment]
Does this still hide the pause when going into PiP? Part of why it was linked to the controls was to automatically hide with them.

**Comment by QuackingCanary** [comment]
Yes, that's the case

**Review by lynxnb — APPROVED** [review]
Nonetheless, code is looking good, thanks!

**Pull request merged**